templates/README.md.tpl

```diff
@@ -57,7 +57,11 @@ First, make sure you [set up your development environment](https://github.com/ge
 
 To add this service to your developer environment:
 ```bash
+{{- if has "python" (stencil.Arg "serviceActivities") }}
+scripts/build-deploy-python-service.sh
```

Comment:
What's the reason for a custom script? The `devenv` supports a `scripts/deploy-to-dev.sh` custom script that could be used instead. The idea was always for `devenv apps deploy` to hide specific scripts/knowledge.

Reply:
As I mentioned in description. Found `devenv apps deploy` is [hard coded on single image creation and deployment](https://outreach-hq.slack.com/archives/C01CAR6GVPD/p1678719039135509?thread_ts=1678316708.202279&cid=C01CAR6GVPD). my script is a wrapper of it, thus a super set to be useful for python developers.

BTW: where is you mentioned `scripts/deploy-to-dev.sh`?  didn't find it in either `devenv` or `devbase` repo. Without seeing it, can't tell whether it can do what I want. Anyway, we need to change a script here in this PR.

Reply:
See Application TypeLegacy (being renamed to `TypeCustom`): https://github.com/getoutreach/devenv/blob/main/internal/app/app.go#L334. 

If that doesn't work for w/e reason -- I'd rather enable custom application build logic for now than remove the abstraction entirely.

Reply:
not quite get you. are you suggesting we rename our script as `scripts/deploy-to-dev.sh` instead? since that file is not existing anywhere and your pointed statement is just a existence detection. It may work if `devenv apps deploy` can invoke the script with our logics automatically. 
My past finding from [your code](https://github.com/getoutreach/devenv/blob/main/internal/app/app.go)  and your make file (under `.bootstrap/root/Makefile` ) can't support more than one images' building and deployments thus made the script to patch.

Reply:
at least for now, even when I renamed the script, it won't be invoked by your code from my understanding.

Reply:
just tried to run `devenv apps deploy .` , proved that it won't invoke `scripts/deploy-to-dev.sh` even when it exists

Reply:
Let's get that fixed then instead of hacking separate instructions in, given it's just a README update then. :D 

I will look into that today.

Reply:
@ygcao can you also link your current script today, just so I can TAL at what it's doing and see if there's potentially some small adjustments I could make to support that?

Reply:
Sure. [here](https://github.com/getoutreach/ds-mlp-website/blob/main/scripts/build-deploy-python-service.sh) it is.
If you can make it built-in, that'll be great! although I feel it's not that straight-forward base on my observation of your codes.